orchestrator/README.md

```diff
@@ -0,0 +1,6 @@
+## orchestrator
+
+This directory stores the script to setup aifabric connection related properties in orchestartor. This script has to be executed throuh powershell in Administrator mode & before running script set execution policy to RemoteSigned by running "Set-ExecutionPolicy RemoteSigned".
+
+For AI Center < 21.4, use orchestratorAutomation.ps1
+For AI Center >= 21.4, use orchestratorAutomationLatest.ps1
```

Comment:
nit: use _21.4 or something instead of latest

Reply:
Actually going forward only this should be used ... but i will confirm naming with jeremy